docs/source/api_reference/regression.rst

```diff
@@ -195,3 +195,14 @@ Base classes
     :template: class.rst
 
     BaseProbaRegressor
+
+Bayesian
+------------
```

Comment:
underline should be as long as the header. I would also put it in a single section with the other regressor.